Extensions/CMake/SlicerBlockAdditionalLauncherSettings.cmake

```diff
@@ -248,6 +248,7 @@ ${EXTENSION_LAUNCHER_SETTINGS_ADDITIONAL_PATH_ENVVARS}
     OUTPUT
       ${Slicer_ADDITIONAL_LAUNCHER_SETTINGS_FILE}
     COMMAND ${CMAKE_COMMAND}
+      -DCMAKE_POLICY_VERSION_MINIMUM:STRING=3.16.3
```

Comment:
```suggestion
```

Instead, the generated script `AdditionalLauncherSettings-configure.cmake` should include the relevant `cmake_minimum_required` call